advisories/unreviewed/2022/05/GHSA-mrr8-fcg7-p2wg/GHSA-mrr8-fcg7-p2wg.json

```
@@ -1,23 +1,52 @@
 {
   "schema_version": "1.3.0",
   "id": "GHSA-mrr8-fcg7-p2wg",
-  "modified": "2022-05-24T17:25:24Z",
+  "modified": "2022-12-17T11:29:57Z",
   "published": "2022-05-24T17:25:24Z",
   "aliases": [
     "CVE-2020-2234"
   ],
-  "details": "A missing permission check in Jenkins Pipeline Maven Integration Plugin 3.8.2 and earlier allows users with Overall/Read access to connect to an attacker-specified JDBC URL using attacker-specified credentials IDs obtained through another method, potentially capturing credentials stored in Jenkins.",
+  "summary": "CSRF vulnerability in Jenkins Pipeline Maven Integration Plugin allow capturing credentials",
```

```suggestion
  "summary": "Missing permission check in Jenkins Pipeline Maven Integration Plugin allow capturing credentials",
```
Little typo, my bad!